packages/database/src/migrations/20210629030806-MapoverlaysForPsssFijiFacilityDemo-modifies-data.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const syndromes = [
+  {
+    dataElement: 'PSSS_AFR_Cases',
+    name: 'AFR',
+    customLabel: 'AFR Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_PF_Cases',
+    name: 'PF',
+    customLabel: 'PF Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_ILI_Cases',
+    name: 'ILI',
+    customLabel: 'ILI Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_DLI_Cases',
+    name: 'DLI',
+    customLabel: 'DLI Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_DIA_Cases',
+    name: 'Diarrhoea',
+    code: 'DIA',
+    customLabel: 'Diarrhoea Case Numbers',
+  },
+];
+
+const syndromeBaseMapOverlay = {
+  userGroup: 'PSSS Tupaia',
+  isDataRegional: true,
+  dataElementCode: 'value',
+  measureBuilder: 'sumLatestPerOrgUnit',
+  countryCodes: '{"FJ"}',
+  projectCodes: '{"psss"}',
+};
+
+const syndromeRadiusMapOverlay = (syndromeName, dateElement, syndromeCode, customLabel) => ({
+  ...syndromeBaseMapOverlay,
+  id: `PSSS_FJ_${syndromeCode}_Syndrome_Bubble_Radius`,
+  name: syndromeName,
+  measureBuilderConfig: {
+    dataElementCodes: [dateElement],
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+    },
+  },
+  presentationOptions: {
+    values: [{ color: 'blue', value: 'other' }],
+    displayType: 'radius',
+    measureLevel: 'Facility',
+    defaultTimePeriod: {
+      unit: 'week',
+      offset: 0,
+    },
+    periodGranularity: 'one_week_at_a_time',
+    customLabel,
+    hideByDefault: {
+      null: true,
+    },
+  },
+});
+
+const syndromeHeatMapOverlay = (syndromeName, dateElement, syndromeCode, customLabel) => ({
+  ...syndromeBaseMapOverlay,
+  id: `PSSS_FJ_${syndromeCode}_Syndrome_Heat_Map`,
+  name: syndromeName,
+  measureBuilderConfig: {
+    dataElementCodes: [dateElement],
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+    },
+  },
+  presentationOptions: {
+    values: [
+      {
+        color: 'grey',
+        value: null,
+      },
+    ],
+    scaleType: 'performanceDesc',
+    displayType: 'shaded-spectrum',
+    measureLevel: 'Facility',
+    defaultTimePeriod: {
+      unit: 'week',
+      offset: 0,
+    },
+    hideByDefault: { null: true },
+    periodGranularity: 'one_week_at_a_time',
+    customLabel,
+  },
+});
+
+const radiusMapOverlayGroup = {
+  id: generateId(),
+  name: 'Syndromic Surveillance Weekly Case Numbers (Radius map)',
+  code: 'FJ_PSSS_Syndromic_Surveillance_Radius_Map_Country',
+};
+const heatMapOverlayGroup = {
+  id: generateId(),
+  name: 'Syndromic Surveillance Weekly Case Numbers (Heat map)',
+  code: 'FJ_PSSS_Syndromic_Surveillance_Heat_Map_Country',
```

Comment:
Hmmm bit of a similar thing here, I think we can rename and re-use the map overlays groups from Palau here as well? ie. `PW_PSSS_Syndromic_Surveillance_Radius_Map_Country` -> `PSSS_Syndromic_Surveillance_Radius_Map_Country`?

The correct overlays should only show up in the appropriate country tho, since the overlays are restricted to the countries in their `countryCodes` field, I believe?

Reply:
Yeah that's right, if I add `FJ` in countryCodes it will show in Fiji.

Reply:
@rohan-bes I think we still need to create new map overlays because they are weekly data, and those for `PW` are daily data.

Reply:
@billli0 Yup, we will need to create new overlays 👍